node-js-web-server/index030.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http');
+const fs = require('fs');
+
+http.createServer(function (req, res) {
+   console.log(req.url, req.method);
+
+});
+
+// Next to read the file you could use readFile method or createReadStream method from fs module.
```

Comment:
This will also have to be broken down into multiple steps like above. You could briefly explain that simple pages have already been written as HTML files, they just need to be served when the user visits a certain URL.

Just go through step-by-step, first with `fs.readFile();`, then build up by having the student add `__dirname`, then `__dirname + '/static/hello.html'`, passing in the callback function, and so on. I know that just running `fs.readFile();` so having them start with writing `fs.readFile('', function() {});` could work, too.